src/montgomery.rs

```diff
@@ -84,6 +110,23 @@ impl CompressedMontgomeryU {
         CompressedEdwardsY(y.to_bytes()).decompress()
     }
 
+    /// Decompress this `CompressedMontgomeryU` to a `MontgomeryPoint`.
+    ///
+    /// Going from affine to projective coordinates, we have:
+    ///
+    ///      u → U/W
+    ///
+    /// # Returns
+    ///
+    /// A projective `MontgomeryPoint` corresponding to this compressed point.
+    pub fn decompress(&self) -> MontgomeryPoint {
+        MontgomeryPoint{
+            // XXX is it a problem here if we're not using a canonical encoding? —isis
```

Comment:
I think the only room for encoding malleability here is from the field elements, and I'm not sure what the problem would be with decoding `2^255 - 18` as `1`... if people need a canonical encoding I guess I think they should use Ristretto?

Reply:
Agreed. Do you think I should remove the comment?

Reply:
Removed the comment in d39e47ff11